Dockerfile.builder

```diff
@@ -0,0 +1,11 @@
+# Make sure the tooling versions are same as defined in the 
```

Comment:
Would suggest to put this file into a subdirectory, we might have scripts associated with this later.